boards/st/stm32l4r9i_disco/stm32l4r9i_disco.dts

```diff
@@ -259,17 +259,16 @@ zephyr_udc0: &usbotg_fs {
 };
 
 &octospi2 {
-	status = "okay";
 	pinctrl-0 = <&octospim_p2_clk_pi6 &octospim_p2_ncs_pg12
 			&octospim_p2_io0_pi11 &octospim_p2_io1_pi10
 			&octospim_p2_io2_pi9 &octospim_p2_io3_ph8
 			&octospim_p2_io4_ph9 &octospim_p2_io5_ph10
 			&octospim_p2_io6_pg9 &octospim_p2_io7_pg10
 			&octospim_p2_dqs_pg15>;
 	pinctrl-names = "default";
+	status = "okay";
 
 	mx25lm51245: ospi-nor-flash@90000000 {
-		status = "okay";
```

Comment:
Equivalent change in *boards/st/stm32l562e_dk/stm32l562e_dk_common.dtsi* is missing.

Reply:
Missed it. Will fix